changelog.d/5862.feature

```diff
@@ -0,0 +1 @@
+[Live location sharing] Improve aggregation process of events
```

Comment:
Maybe use `.wip` extension

Reply:
Thanks! I missed there was an extension for non-finished feature.